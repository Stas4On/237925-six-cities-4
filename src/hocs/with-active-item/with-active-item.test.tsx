import * as React from "react";
import * as renderer from "react-test-renderer";
import withActiveItem from "./with-active-item";

const MockComponent = (props) => {
  const {children} = props;

  return (
    <div>
      {children}
    </div>
  );
};

const MockComponentWrapped = withActiveItem(MockComponent);

it(`should render HOC withActiveIndex correct`, () => {
  const tree = renderer
      .create((
        <MockComponentWrapped activeItemId = {0}/>
      ), {
        createNodeMock() {
          return {};
        }
      })
      .toJSON();

  expect(tree).toMatchSnapshot();
});
