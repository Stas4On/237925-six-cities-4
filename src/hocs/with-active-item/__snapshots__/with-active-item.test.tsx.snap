// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render HOC withActiveIndex correct 1`] = `<div />`;
